build-system/tasks/build.js

```diff
@@ -95,10 +95,10 @@ async function performBuild(watch, defaultTask) {
     parseExtensionFlags();
   }
   await Promise.all([compileCss(watch), compileJison()]);
-  await Promise.all([
-    bootstrapThirdPartyFrames(watch),
-    compileCoreRuntime(watch),
-  ]);
+  await bootstrapThirdPartyFrames(watch);
```

Comment:
Pretty sure `bootstrapThirdPartyFrames(watch)` can be done in parallel with `compileCss(watch)` and `compileJison()` in the line above.